pkg/scheduler/plugins/rescheduling/node_utilization_util.go

```diff
@@ -61,8 +61,8 @@ func getNodeUtilization() []*NodeUtilization {
 		nodeUtilization := &NodeUtilization{
 			nodeInfo: nodeInfo.Node,
 			utilization: map[v1.ResourceName]float64{
-				v1.ResourceCPU:    nodeInfo.ResourceUsage.CPUUsageAvg[Interval],
-				v1.ResourceMemory: nodeInfo.ResourceUsage.MEMUsageAvg[Interval],
+				v1.ResourceCPU:    nodeInfo.ResourceUsage.CPUUsageAvg[MetricsPeriod],
```

Comment:
I understand that the analysis of MetricsPeriod should be added when config is analyzed, stored in the Configs structure, and then used here.

Will this implementation take effect by configuring MetricsPeriod through the yaml file?

Reply:
@wangyang0616 Fixed